src/containers/Nav/Nav.tsx

```diff
@@ -1,14 +1,21 @@
 import Toolbar from '@material-ui/core/Toolbar'
 import React from 'react'
+import { useIsDesktop } from '../../styles'
 import * as S from './Nav.styles'
 import { NavItems } from './NavItems'
+import { NotificationBar } from './NavNotificationBar'
 
 export function Nav() {
+  const isDesktop = useIsDesktop()
+
   return (
     <S.Grow>
       <S.AppBar
         position="static"
       >
+        {
+          isDesktop && <NotificationBar />
```

Comment:
Favorise plutôt les ternaires dans du JSX. Exemple:
`isDesktop ? <MyComponent/> : undefined`
source : https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

Reply:
Pas convaincu tant que ça, c'est valable uniquement pour un tableau.

Reply:
@GuillaumeJasmin, c'est plus par "habitude"

Reply:
Je trouve ça plus propre cette forme là. Après ça dépend des cas, mais des-fois je mets le `!!` devant pour être sûr que ça devienne un booléen.